content/posts/newsletter-019/index.md

```diff
@@ -526,19 +522,124 @@ components update, wireframes and mouse ray modules.
+This ensures that when an asset like a mesh is loaded, other related assets like
+textures/material/vertex data are loaded. By integrating with Distill, rafx
+provides advanced features like streaming live asset updates to remote devices.
+
+More information about rafx:
+
+- [Github][rafx-github]
+- [Documentation][rafx-documentation]
+- [Why Rafx?][why-rafx] (includes similarities/differences with other rust and
+  non-rust alternatives)
+
+[rafx-github]: https://github.com/aclysma/rafx
+[rafx-documentation]: https://github.com/aclysma/rafx/blob/master/docs/index.md
+[why-rafx]: https://github.com/aclysma/rafx/blob/master/docs/why_rafx.md
+[rafx-api-design]: https://github.com/aclysma/rafx/blob/master/docs/api/api_design_in_rust_psuedocode.rs
+[rafx-api-triangle-example]: https://github.com/aclysma/rafx/blob/master/rafx/examples/api_triangle/api_triangle.rs
+[rafx-gdc-2015]: http://advances.realtimerendering.com/destiny/gdc_2015/Tatarchuk_GDC_2015__Destiny_Renderer_web.pdf
+[rafx-gdc-2017]: https://www.gdcvault.com/play/1024612/FrameGraph-Extensible-Rendering-Architecture-in
+[rafx-distill]: https://github.com/amethyst/distill
+
+### [egui]
+
+![The plot thickens](egui-plot.gif)
+
+[egui] by [@emilk] is an easy-to-use immediate mode GUI library in pure Rust.
+
+This month [versions 0.9 and 0.10] of egui were released with many improvements
+big and small, including a 2D plot, more text styles, disabling widgets and
+improved documentation.
+
+You can try out egui in the [online demo].
+
+[egui]: https://github.com/emilk/egui
+[online demo]: https://emilk.github.io/egui
+[versions 0.9 and 0.10]: https://github.com/emilk/egui/blob/master/CHANGELOG.md
+[@emilk]: https://twitter.com/ernerfeldt
+
+### [This Month in Mun][mun-february]
+
+[![Mun logo](mun-logo.png)][Mun]
+
+[Mun] is a scripting language for gamedev focused on quick iteration times that
+is written in Rust.
+
+It's been a long time coming, but the Mun Core Team is closing in on the finish
+line for Mun v0.3. They are only a couple of pull requests away from locking the
+build for bug fixes and documentation. The [February updates][mun-february]
+include:
+
+- `use` statements language support;
+- Incremental file updates for the language server;
+- LLVM 11 support;
+- Bug fixes and other improvements.
+
+[Mun]: https://mun-lang.org
+[mun-february]: https://mun-lang.org/blog/2021/03/04/this-month-february
+
+### [Graphite][graphite-repo]
+
+![Graphite GUI](graphite.png)
+_Progress on the GUI_
+
+[Graphite][graphite-repo] ([GitHub][graphite-repo], [Discord][graphite-discord],
+[Twitter](https://twitter.com/GraphiteEditor)) is an in-progress vector and
+raster graphics editor built on a nondestructive node-based workflow.
+
+Since February's Rust Gamedev Meetup [which announced][graphite-video] the
+Graphite vision and attracted tremendous interest, community advise has shifted
+the development strategy to focus on a 0.1 MVP release ASAP:
```

Comment:
```suggestion
Since February's Rust Gamedev Meetup [which announced][graphite-video] the
Graphite vision has attracted tremendous interest, community advice has shifted
the development strategy to focus on a 0.1 MVP release ASAP:
```